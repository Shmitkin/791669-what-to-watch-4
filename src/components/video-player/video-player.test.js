import React from "react";
import renderer from "react-test-renderer";

import VideoPlayer from "./video-player.jsx";


it(`Should VideoPlayer render correctly`, () => {

  const tree = renderer
    .create(
        <VideoPlayer
          src = {`src/src/src`}
          poster = {`img/someimage`}
          isPlaying = {true}
          muted = {true}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
