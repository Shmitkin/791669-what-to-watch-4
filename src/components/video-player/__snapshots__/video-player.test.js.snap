// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should VideoPlayer render correctly 1`] = `
<video
  className=""
  height="175"
  loop={true}
  muted={true}
  poster="img/someimage"
  src="src/src/src"
  width="280"
/>
`;
